### ISSUE TITLE
possibility to close issues

### ISSUE THREAD

**Issue by lassnj** [center-for-learning-management/moodle-local_edusupport#18]
Managers cannot close issues in a forum. They should have a button to close the issue. The forum entry should then be moved to another list as it gets confusing if you keep collecting issues.

**Comment by rschrenk** [comment]
Moodle has a default functionality for closing discussion threads. We do not need to implement this ourselves.

**Comment by lassnj** [comment]
Yes, but they don't have the right to do so as the option "Lock this discussion" is not available.

**Comment by rschrenk** [comment]
This permission is given on course level, not by this plugin. We have to ensure that the editing teacher role has this capability.

**Comment by lassnj** [comment]
Find Workaround as there is no special capability for closing issues
